src/libraries/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -77,8 +77,8 @@ public static Task CreateClientAndServerAsync(Func<Uri, Task> clientFunc, Func<L
         {
             return CreateServerAsync(async server =>
             {
-                Task clientTask = clientFunc(server.Address);
-                Task serverTask = serverFunc(server);
+                Task clientTask = Task.Run(() => clientFunc(server.Address));
+                Task serverTask = Task.Run(() => serverFunc(server));
```

Comment:
I'm curious about this change. Is the purpose to reduce deadlocks when testing against the sync client APIs?

The change here appears to push off the client and server tasks to background tasks.

Reply:
Removed, this came from the original prototype and it's not necessary now, thanks for spotting this :+1: